front-end/src/app/shared/components/contact-lookup/contact-lookup.component.ts

```diff
@@ -53,13 +55,16 @@ export class ContactLookupComponent {
     ])
   );
 
+  selectedFecCommitteeAccount: CommitteeAccount | undefined = undefined;
```

Comment:
`selectedFecCommitteeAccount: CommitteeAccount | undefined;` is equivalent i believe.